interfaces/prompting/requestrules/requestrules.go

```diff
@@ -0,0 +1,1083 @@
+package requestrules
```

Comment:
in a follow up, it would be good to add package doc comment to all the 3 packages, prompting, promting/request(rules|prompts)

Reply:
Perhaps we could just add simple ones here to avoid the problem with the copyright being treated as package doc comment.